backstop/tests/button.html

```diff
@@ -1392,29 +1396,56 @@ <h3>Input + button combo group</h3>
 
     <section id="demo-combo-group">
       <div class="iui-button-group">
-        <div>
+        <div class="iui-button-group-item">
           <button class="iui-button iui-default"><span class="iui-button-label">Button 1</span></button>
         </div>
-        <div>
+        <div class="iui-button-group-item">
           <input class="iui-input" />
         </div>
-        <div>
+        <div class="iui-button-group-item">
           <button class="iui-button iui-default"><span class="iui-button-label">Button 2</span></button>
         </div>
       </div>
 
       <br />
 
       <div class="iui-button-group">
-        <div>
+        <div class="iui-button-group-item">
           <div class="iui-input-container">
             <input
               class="iui-input"
               value="https://itwin.github.io/iTwinUI"
             />
           </div>
         </div>
-        <div><button class="iui-button iui-high-visibility">Copy</button></div>
+        <div class="iui-button-group-item"><button class="iui-button iui-high-visibility">Copy</button></div>
+      </div>
+
+      <br />
+
+      <div class="iui-button-group">
```

Comment:
I dont know, maybe I just start to be picky on not related things, but is this focus style supposed to be like that?
![image](https://user-images.githubusercontent.com/81580355/151154527-d0d3d68f-7cb2-4b81-9562-cbce5e4d098d.png)
![image](https://user-images.githubusercontent.com/81580355/151154544-76d0b559-f8d0-456b-8e20-0310dd8d8601.png)
![image](https://user-images.githubusercontent.com/81580355/151154556-47a19d26-2cd6-42d6-9383-63457270c223.png)

Notice different border on the close icon. Is it the expected styling for bordered vs borderless components?
@FlyersPh9

Reply:
Yes it is expected to be like this and has nothing to do with this PR
- Bordered components go from 1px border to 2px outline
- Borderless components go from no border to 1px outline

Reply:
Ok, I know its not related, that why I said in my comment "not related things".
Just wanted to confirm, resolving